released/packages/coq-parseque/coq-parseque.0.1.1/opam

```diff
@@ -0,0 +1,24 @@
+opam-version: "2.0"
+maintainer: "G. Allais"
+dev-repo: "git+https://github.com/gallais/parseque"
+bug-reports: "https://github.com/gallais/parseque/issues"
+homepage: "https://github.com/gallais/parseque"
+authors: ["G. Allais"]
+license: "GPL-3.0"
```

Comment:
Is this supposed to be `GPL-3.0-only` or `GPL-3.0-or-later`? Unfortunately, unqualified `GPL-3.0` is ambiguous. (cc: @gallais)

Reply:
I'm happy with `-or-later`.